### PR TITLE
feat: move icons to icons.md from readme.md

### LINKED ISSUE #1195: [DOCS] Move the supported Icons to a seprate file.
### Description

- Move the [Available Icons](https://github.com/EddieHubCommunity/LinkFree#available-icons) section from the README to a file name `icons.md`

- Also, add a message in the README for the contributors to check the available icon file.
It can something like this, and hyperlink the icon's file.
_"Check [`Icons.md`]() for the all the supported icons"_

### Screenshots

Section to be moved
Screenshot from 2022-03-23 09-59-56

### PR DESCRIPTION
**Pull request by kishoreandra**
Move icons from readme.md to a separate file (icons.md) and add link to navigate to the same from readme.

Fixes #1195 

## Changes proposed

- [x]  Move icons to seperate file; marked as done

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1200"></a>

### CODE REVIEW

File: README.md
Comment (schmelto) [inline README.md:110]:
```suggestion
Check [Icons.md](/icons.md) for all the supported icons.
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]
Lgtm

**Review by vikasganiga05 — APPROVED** [review]
LGTM :+1: Thanks! for your contribution :tada:

**Review by ssensalo — APPROVED** [review]

**Comment by kishoreandra** [comment]
@vikasganiga05 @schmelto 
Great 😃 Thank you

**Comment by schmelto** [comment]
> @vikasganiga05 @schmelto 
> 
> Great 😃 Thank you, what would be the next steps 🤔 

Waiting for another maintainer to approve and than the pr will get merged. No need from you side to do something.

**Comment by kishoreandra** [comment]
> > @vikasganiga05 @schmelto
> > Great 😃 Thank you, what would be the next steps 🤔
> 
> Waiting for another maintainer to approve and than the pr will get merged. No need from you side to do something.

Awesome 😎

**Review by emmalearnscode — APPROVED** [review]
Looks good, thanks for contributing!

**Comment by kishoreandra** [comment]
> Looks good, thanks for contributing!

Thank you 😊, my pleasure

**Pull request merged**
